### ISSUE TITLE
playback error

### ISSUE THREAD

**Issue by nammminn2** [Wilson-Magic/plugin.video.hummingbird#20]
was trying to watch an episode of a show it loaded but then it gave a playback error
https://streamable.com/58k3bb

**Comment by tripod** [comment]
I too am having a playback issue. It finds the sources, just refuses to play them if you need other information I'm happy to oblige.
Here is the full log.
http://sprunge.us/hv8lSy

**Comment by Goldenfreddy0703** [comment]
Hey I love the auramod. Ok let me take a look at this issue and I will see for myself.

**Comment by Goldenfreddy0703** [comment]
Hey guess what, the addon doesn't work for me so that probably means it under maintenance at the moment but hey Wison could probably resolve this issue, I heard he is going to make some major updates for this addon in the future so dont worry.

**Comment by AdminZo21** [comment]
I have the same problem. I was trying to watch "Black Clover" and tried episodes from 1 to 8 and none of them play. It just gives me 2 sources:
1080 | AnimePahe | Kwik.cx | Sub
720 | AnimePahe | Kwik.cx | Sub

Here is the log (I installed Log Viewer for Kodi and this is what it shows me after I try to play the link):

2020-04-26 12:01:55.179 T:9519 ERROR: EXCEPTION Thrown (PythonToCppException): -->Python callback/script returned the following error<--
-NOTE: IGNORING THE CAN LEAD TO MEMORY LEAKS!
Error type: <class 'cloudflare_exceptions.Cloudflate_reCaptcha_Provider'>
Error Contents: Cloudflare reCaptcha detected, unfortunately you haven't loaded an anti reCaptcha provider correctly via the 'recaptcha' parameter.
Traceback (most recent call last): Cloudflare_reCaptcha_Provider: Cloudflare reCaptcha detected, unfortunately you haven't loaded an anti reCaptcha provider correctly via the 'recaptcha' parameter.
-->End of Python script error report<--

**Comment by Wilson-Magic** [comment]
Hi all, Sorry for the late reply.

@nammminn2 can you get a log for me? I can't tell what's causing that without it. Also disable AnimePahe. It's broken right now.

@tripod That is an issue where you don't have one of the dependencies installed. It's Pycryptodome. It's needed for AnimeTwist. It should be available out the box assuming you are on Kodi 18.

@Goldenfreddy0703 Can you log me up please? If nothing is working it's probably something pretty bad.

**Comment by Goldenfreddy0703** [comment]
Holy crap I thought this addon got abandoned. Can you tell all the other people that your addon it working, I've been trying to reply to them and I accedently said your addon was not working becouse I thought it wasn't tbh. I'm so sorry man for saying it was not working and I'm glad to see you back again. The only thing people just need to do on your addon is to put input stream adaptive to default settings, and disabling smart play, and make the playback auto select, one last thing is to add a wonderful subs and that's all ,and also disabling fanart tv setting. I'm glad to see you back again and I'm sorry for the false comments on some of the issues, that was my fault and I'm sorry.

**Comment by AdminZo21** [comment]
> Hi all, Sorry for the late reply.
> 
> @nammminn2 can you get a log for me? I can't tell what's causing that without it. Also disable AnimePahe. It's broken right now.
> 
> @tripod That is an issue where you don't have one of the dependencies installed. It's Pycryptodome. It's needed for AnimeTwist. It should be available out the box assuming you are on Kodi 18.
> 
> @Goldenfreddy0703 Can you log me up please? If nothing is working it's probably something pretty bad.

Hi.

I have the same problem. I was trying to watch "Black Clover" and tried episodes from 1 to 8 and none of them play. It just gives me 2 sources:
1080 | AnimePahe | Kwik.cx | Sub
720 | AnimePahe | Kwik.cx | Sub

Here is the log (I installed Log Viewer for Kodi and this is what it shows me after I try to play the link):

2020-04-26 12:01:55.179 T:9519 ERROR: EXCEPTION Thrown (PythonToCppException): -->Python callback/script returned the following error<--
-NOTE: IGNORING THE CAN LEAD TO MEMORY LEAKS!
Error type: <class 'cloudflare_exceptions.Cloudflate_reCaptcha_Provider'>
Error Contents: Cloudflare reCaptcha detected, unfortunately you haven't loaded an anti reCaptcha provider correctly via the 'recaptcha' parameter.
Traceback (most recent call last): Cloudflare_reCaptcha_Provider: Cloudflare reCaptcha detected, unfortunately you haven't loaded an anti reCaptcha provider correctly via the 'recaptcha' parameter.
-->End of Python script error report<--

**Comment by b1n4ryk1lla** [comment]
@AdminZo21 try disabling animephae in settings seems there's alot of issues with that source should work after that

**Comment by AdminZo21** [comment]
> @AdminZo21 try disabling animephae in settings seems there's alot of issues with that source should work after that 

did that and now it gives me no sources. (tried with Black Clover)

One some anime it gives me only 1 source:

480 | AnimeTwist | Twist.moe | Sub.

Either this or nothing.

Thank you for the tip, though.

**Comment by Wilson-Magic** [comment]
Yeah animepahe is messed up and I can't really fix it, I'm looking into seeing what I can do to make alternative means of scraping.
